src/App/Tracker/Controller/Hooks/Listeners/JoomlacmsPullsListener.php

```diff
@@ -231,6 +244,12 @@ class JoomlacmsPullsListener extends AbstractListener
 			'^administrator/components/com_fields',
 			'^components/com_fields',
 		],
+		// Composer Change
+		'72' => [
```

Comment:
These 'magic numbers' (the array keys)  should be replaced with constants (or strings) to increase readability and maintainability.